src/gc/impl/proto/gc.d

```diff
@@ -1,8 +1,7 @@
 
 module gc.impl.proto.gc;
 
-import gc.config;
-import gc.gcinterface;
+import core.gc.gcinterface;
```

Comment:
Couldn't this file import the declarations from `gc.proxy` instead of defining the headers again?

Reply:
Not sure what you mean. Do you prefer a cyclic dependency and a public import in gc.proxy over a simple import of the public module?

Reply:
> Do you prefer a cyclic dependency

Ah sorry. Nevermind then.

Reply:
Ah, I didn't see the gc_* declarations in the diff. I think we have these declarations elsewhere too, e.g. core.memory. IIRC this is done to fake some attributes.